akka-distributed-data/src/main/scala/akka/cluster/ddata/GSet.scala

```diff
@@ -4,15 +4,17 @@
 package akka.cluster.ddata
 
 object GSet {
-  private val _empty: GSet[Any] = new GSet(Set.empty)
+  private val _empty: GSet[Any] = new GSet(Set.empty)(None)
   def empty[A]: GSet[A] = _empty.asInstanceOf[GSet[A]]
   def apply(): GSet[Any] = _empty
+  def apply[A](set: Set[A]): GSet[A] = new GSet(set)(None)
+
   /**
    * Java API
    */
   def create[A](): GSet[A] = empty[A]
```

Comment:
I have hidden the Scala `apply(set)` instead, as per discussion on gitter. If MiMa passes, I will squash the commit (my day-time rig exhausts RAM on MiMa checks, sorry)